app/(main)/profile/page.tsx

```diff
@@ -267,7 +276,7 @@ export default function ProfilePage() {
             <AddressLink
               href={getExplorerAddressUrl(address)}
               address={address}
-              isAIAgent={isAIAgent}
+              aiAgent={aiAgent as AiAgent | undefined}
```

Comment:
Let's return the type on the `getIsAIAgent` function.

Reply:
I tried to do that but the return type on that `useMemo` fn is different, it returns:
`Record<string, unknown> | undefined`

But I don't know why it returns a record with `unknown` in that type, that's why I casted it.